collect_app/src/main/java/org/odk/collect/android/projects/QrCodeProjectCreatorDialog.kt

```diff
@@ -126,7 +126,9 @@ class QrCodeProjectCreatorDialog :
             requireActivity(),
             object : PermissionListener {
                 override fun granted() {
-                    startScanning(savedInstanceState)
+                    if (isAdded) {
```

Comment:
I'm not sure if there is a good way to add a test for this (it would be good to experiment if it can be simulated with `FragmentScenario`) but if not I think we should at least have comment on why we're doing this.

Reply:
I added a comment since I don't think there is a good way to test it. We would need to UIAutomator to accept that permission because espresso can't access components from outside of our application’s package (like those permission dialogs).

Reply:
Yeah fair enough!